### PR TITLE
Build issues

### PR DESCRIPTION
**Pull request by msaxena2**
Issues that deal with the build failing on OS X. @dwightguth please review.

### CODE REVIEW

File: semantics/Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ c11-kompiled/c11-kompiled/def.$(EXTENSION): $(C11_FILES)
 
 c11-nd-kompiled/c11-nd-kompiled/def.$(EXTENSION): $(C11_FILES)
 	@echo "Kompiling the dynamic C semantics with expression sequencing non-determinism..."
-	$(KOMPILE) --backend ocaml --non-strict --smt none c11.k -d "c11-nd-kompiled" --transition "observable ndtrans" --superheat "ndheat" --supercool "ndlocal" -I $(PROFILE_DIR)/semantiocs
+	$(KOMPILE) --backend ocaml --non-strict --smt none c11.k -d "c11-nd-kompiled" --transition "observable ndtrans" --superheat "ndheat" --supercool "ndlocal" -I $(PROFILE_DIR)/semantics
```
Comment (msaxena2) [inline semantics/Makefile]: This I believe is a typo. @dwightguth please confirm.

Comment (dwightguth) [inline semantics/Makefile]: yup. I haven't built the full nondeterministic version of the semantics in a while, which is why this wasn't caught.

File: Makefile [hunk 2]
```diff
@@ -41,21 +41,21 @@ $(DIST_DIR)/kcc: $(FILES_TO_DIST) $(wildcard $(PROFILE_DIR)/include/*) $(PROFILE
...
-	@cp -p -r $(SEMANTICS_DIR)/c11-kompiled $(DIST_DIR)/$(PROFILE)
+	@cp -p -RL $(SEMANTICS_DIR)/c11-kompiled $(DIST_DIR)/$(PROFILE)
 
 $(DIST_DIR)/$(PROFILE)/c11-translation-kompiled/c11-translation-kompiled/def.$(EXTENSION): $(DIST_DIR)/kcc translation-semantics
-	@cp -p -r $(SEMANTICS_DIR)/c11-translation-kompiled $(DIST_DIR)/$(PROFILE)
+	@cp -p -RL $(SEMANTICS_DIR)/c11-translation-kompiled $(DIST_DIR)/$(PROFILE)
 
 $(DIST_DIR)/$(PROFILE)/c11-nd-kompiled/c11-nd-kompiled/def.$(EXTENSION): semantics
-	@cp -r $(SEMANTICS_DIR)/c11-nd-kompiled $(DIST_DIR)/$(PROFILE)
+	@cp -RL $(SEMANTICS_DIR)/c11-nd-kompiled $(DIST_DIR)/$(PROFILE)
```
Comment (dwightguth) [inline Makefile]: I think these should probably be -p as well otherwise you might rebuild them when nothing's really changed.

File: Makefile [hunk 3]
```diff
@@ -41,21 +41,21 @@ $(DIST_DIR)/kcc: $(FILES_TO_DIST) $(wildcard $(PROFILE_DIR)/include/*) $(PROFILE
...
 
 $(DIST_DIR)/$(PROFILE)/c11-nd-thread-kompiled/c11-nd-thread-kompiled/def.$(EXTENSION): semantics
-	@cp -r $(SEMANTICS_DIR)/c11-nd-thread-kompiled $(DIST_DIR)/$(PROFILE)
+	@cp -RL $(SEMANTICS_DIR)/c11-nd-thread-kompiled $(DIST_DIR)/$(PROFILE)
```
Comment (dwightguth) [inline Makefile]: also here

**Comment by dwightguth** [comment]
looks good. I am just going to go ahead and merge this and I will fix the comment I made about -p in my next pull request which reorganizes the build system.

**Pull request merged**